packages/terra-props-table/src/PropsTable.scss

```diff
@@ -0,0 +1,5 @@
+:local {
+    .props_table_pre {
```

Comment:
The style naming convention our team follows is using hyphenated class names. This should be updated to:
`.props-table-pre {`